CHANGES.txt

```diff
@@ -1,6 +1,14 @@
 Unreleased
 ----------
 
+Experimental Features
+~~~~~~~~~~~~~~~~~~~~~
+
+These features are experimental and may change at any point in the future.
+
+- The cookie API's now have the ability to set the SameSite attribute in both
+  webob.cookies.make_cookie and webob.cookies.CookieProfile.
```

Comment:
No, this is in CHANGES.txt, I haven't marked any of the other things up either. I might add `` around them.